crates/bevy_pbr/src/render/skin.rs

```diff
@@ -232,3 +233,18 @@ pub fn no_automatic_skin_batching(
         commands.entity(entity).try_insert(NoAutomaticBatching);
     }
 }
+
+pub fn mark_meshes_as_changed_if_their_skins_changed(
```

Comment:
For consistency:
```suggestion
pub fn mark_3d_meshes_as_changed_if_their_skins_changed(
```

It would be nice to port all this to 2D as well... :)